go.mod

```diff
@@ -5,7 +5,6 @@ go 1.21.6
 require (
 	github.com/edaniels/golog v0.0.0-20230215213219-28954395e8d0
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.19.0
-	go.opentelemetry.io/otel/sdk v1.22.0
```

Comment:
This wasn't used in Go module and was conflicting with the `resource` reference in from the RDK